.travis.yml

```diff
@@ -1,17 +1,9 @@
-language: cpp
-sudo: required
-dist: trusty
-compiler:
-    - clang
-os:
-    - osx
-osx_image: xcode9.3
-ruby: 2.2.0
-before_install:
-    - if [ $TRAVIS_OS_NAME == osx ]; then rm '/usr/local/include/c++' && brew update && brew upgrade python && brew install root hdf5 fftw libmatio ; fi
+language: ruby
+services:
+- docker
 script:
-    - if [ $TRAVIS_OS_NAME == osx ]; then source CI/build.sh ; fi
-    - if [ $TRAVIS_OS_NAME == osx ]; then source CI/test.sh ; fi
+- docker build -t katydid --build-arg build_type=DEBUG --build-arg build_tests_exe=TRUE .
+- docker run -v /home/travis/build/project8/katydid:/katydid katydid /bin/sh -c "source /usr/local/p8/katydid/v2.15.2/setup.sh; source /usr/local/p8/katydid/v2.15.2/bin/this_cicada.sh; cd /katydid/CI; bash test.sh"
```

Comment:
This has hard-coded version numbers, which will require keeping up-to-date every time there's a release.  We need to find a way to avoid doing this.

Reply:
Solved by using the Katydid_build_prefix link